### DISCUSSION TITLE
update not working with python 3.8

### DISCUSSION THREAD

**Discussion by bastieb** [pretix/pretix#2129] [Q&A]
I recently updated python for pretix, because there were some problems with the update. Now I have two folders under /var/pretix/venv/lib: python3.7 and python3.8 and I just get one small error during the python rebuild:
 `The po files under /var/pretix/venv/lib/python3.8/site-packages/pretix/locale/nl_Informal/LC_MESSAGES are in a seemingly not writable location. mo files will not be updated/created.`

But I think that should not bother. But after I finished the update process and restarted the pretix-worker and  pretix-web there is no updated version on my server. Is it a Bug with python3.8 or do I did something wrong?

**Comment by raphaelm** [comment]
pretix works fine with python 3.8. The error message you posted indicates that you maybe installed something as root once and then as pretixuser again? Please make sure that all files in /var/pretix belong to the user pretix runs at. Also, check your service file taht it runs the correct path.
